src/modules/overlay/actions.js

```diff
@@ -48,9 +55,8 @@ export function closeOverlay({ commit, state }, { id, transition } = {}) {
 
         commit(PREPARE_CLOSE_OVERLAY, { id, transition });
 
-        Vue.nextTick(() => {
+        setTimeout(() => {
```

Comment:
We can use an helper function instead of setTimeout, You can try something similar https://github.com/developit/preact/blob/master/src/util.js#L30

Reply:
done